client/src/components/Navbar/style.module.scss

```diff
@@ -3,9 +3,17 @@
   justify-content: space-around;
   align-items: center;
   padding: 20px;
-  border-top: 1px solid gray;
+  box-shadow: 0px -1px 3px 0px #d9d9d9;
 }
 
 .navbarActive {
   color: green;
 }
+
+.button {
+  border: 0;
+  padding: 0;
+  background-color: white;
```

Comment:
let's not use color keywords (white, black etc.) because they may differ across OS/browsers
change to HEX (#fff)

Reply:
feel free to change it in next PR (along with setting explicitly color for navbar buttons and Hamburder)